lib/src/Uncertainty/Algorithm/MetaModel/Kriging/openturns/GeneralizedLinearModelAlgorithm.hxx

```diff
@@ -162,15 +162,86 @@ protected:
 
 private:
 
+  // Helper class to compute the reduced log-likelihood function of the model
+  class ReducedLogLikelihoodEvaluation: public NumericalMathEvaluationImplementation
+  {
+  public:
+    // Constructor from a GLM algorithm
+    ReducedLogLikelihoodEvaluation(GeneralizedLinearModelAlgorithm & algorithm)
+      : NumericalMathEvaluationImplementation()
```

Comment:
Each call to ReducedLogLikelihoodEvaluation::operator() modify the internal state of the GLMAlgo through mutable attributes, even if the method is const. You need to get back these changes, so even if the method is declared to be const I prefer not to qualify the given reference as const, as the object IS modified.